libs/common/src/sql-db-connection/custom-naming-strategy.test.ts

```diff
@@ -0,0 +1,60 @@
+import { CustomNamingStrategy, firstLetterCase } from './custom-naming-strategy';
+
+describe('custom-naming-strategy', () => {
+  describe('CustomNamingStrategy', () => {
+    const strategy = new CustomNamingStrategy();
+
+    describe('columnName', () => {
+      describe('when customName is provided', () => {
+        it.each`
+          customName       | embeddedPrefixes          | expected                  | description
+          ${'custom'}      | ${[]}                     | ${'custom'}               | ${'there are no embedded prefixes'}
+          ${'custom'}      | ${['prefix']}             | ${'prefixCustom'}         | ${'there is one embedded prefixes'}
+          ${'custom'}      | ${['prefix1', 'prefix2']} | ${'prefix1Prefix2Custom'} | ${'there are multiple embedded prefixes'}
+          ${'customName'}  | ${['prefixWord']}         | ${'prefixWordCustomName'} | ${'the inputs are in camel case'}
+          ${'custom_name'} | ${['prefix_word']}        | ${'prefixWordCustomName'} | ${'the inputs are in snake case'}
+          ${'custom name'} | ${['prefix word']}        | ${'prefixWordCustomName'} | ${'the inputs are in space separated'}
+        `(
+          'should return the customName prefixed by all the prefixes in camel case $description',
```

Comment:
Should this be the following?
```suggestion
          'should return the customName prefixed by all the prefixes in camel case when $description',
```

Reply:
Same for the test case below this (line 36 at present)